org.opent2t.sample.thermostat.superpopular/org.opent2t.sample.thermostat.superpopular.json

```diff
@@ -91,6 +91,18 @@
       "$ref": "oic.baseResource.json#/definitions/oic.r.baseresource"
     },
     {
+      "$ref": "oic.r.mode.json#/definitions/oic.r.mode"
+    },
+    {
+      "$ref": "oic.r.clock.json#/definitions/oic.r.clock"
+    },
+    {
+      "$ref": "oic.r.temperature.json#/definitions/oic.r.temperature"
+    },
+    {
+      "$ref": "oic.r.humidity.json#/definitions/oic.r.humidity"
+    },
```

Comment:
See the Airflow Control for an example:

[http://oneiota.org/revisions/1485](http://oneiota.org/revisions/1485)

---
In reply to: [79683703](https://github.com/openT2T/translators/pull/102#discussion_r79683703) [](ancestors = 79683703,79681760,79676179)